app/components/application_record_card/view.rb

```diff
@@ -48,6 +49,12 @@ def trainee_id
       tag.p("Trainee ID: #{record.trainee_id}", class: "govuk-caption-m govuk-!-font-size-16 application-record-card__id govuk-!-margin-top-0 govuk-!-margin-bottom-1")
     end
 
+    def provider_name
+      return unless system_admin
+
+      tag.p("Provider: #{record.provider.name}", class: "govuk-caption-m govuk-!-font-size-16 application-record-card__provider_name govuk-!-margin-top-0 govuk-!-margin-bottom-1")
```

Comment:
```suggestion
      tag.p(record.provider.name, class: "govuk-caption-m govuk-!-font-size-16 application-record-card__provider_name govuk-!-margin-top-0 govuk-!-margin-bottom-1")
```
I think we don't need the "Provider:" bit?

Reply:
![image](https://user-images.githubusercontent.com/470137/140302123-b25734b4-5bd3-41a5-a1e3-e1c32eab570a.png)

Its that bit...

Reply:
Actually you was right :heart: